tools/runner

```diff
@@ -209,6 +209,14 @@ for key in libs.keys():
 
 test_params['defines'] = test_params['defines'].split()
 
+# Some tests have *.vlt and *.cc files which are specific to verilator,
+# so filter them out for all other runners.
+if 'verilator' not in args.runner:
+    test_params['files'] = [
+        f for f in test_params['files']
+        if not f.endswith('.vlt') and not f.endswith('.cc')
+    ]
+
```

Comment:
Hmm, we have indeed allowed similar hacks in some of the wrappers for specific runners, but this is a common file for all runners and I'd rather keep it cleaner (i.e. it should not be aware of any particular runners).

Maybe instead of this, we could create an "allow list" of file extensions. The base runner could then define the default allow list as `[".sv", ".v", ".vh", ".svh"]` (and others if relevant).

The verilator runners could then extend the list with `.vlt` and `.cc`.

And here, instead of your original change, we could filter out the files list based on the allow list.

Reply:
Sure, I can look at that.